backend/test/test-support/shared/core/TeamIdGeneratorStub.ts

```diff
@@ -0,0 +1,10 @@
+import { EntityIdGenerator } from '../../../../src/shared/core/application/EntityIdGenerator';
+
+export function TeamIdGeneratorStub(idsList: string[]): EntityIdGenerator {
+  idsList.reverse();
+  return {
+    generate(): string {
+      return <string>idsList.pop();
```

Comment:
Sprytna implementacja :) Fajnie to jakos inaczej nazwac, bo jak widze nie generuje to tylko TeamId, ale jakiekolwiek.